src/TulsiGenerator/PBXObjects.swift

```diff
@@ -236,6 +236,15 @@ final class PBXFileReference: PBXReference, Hashable {
     } else if let uti = explicitFileType {
       try serializer.addField("explicitFileType", uti)
     }
+
+    // Give Starlark files Python syntax highlighting by default.
+    // It's quite a good match--and certainly a better default than no highlighting.
+    // Xcode's plugin functionality is currently so limited that we can't do better.
+    if name == "BUILD" || name == "WORKSPACE" || name.hasSuffix(".bzl") {
```

Comment:
would also be nice to check the `.bazel` suffix, since many people use `BUILD.bazel` for their build files